docs/en/client-protocol/client.md

```diff
@@ -0,0 +1,127 @@
+---
```

Comment:
General comment - can we call this "native protocol "not "client protocol"

Reply:
path should reflect this as well

Reply:
So no new naming for Native protocol, let's stick with native?

Reply:
So lots of terms floating about:

1. Native protocol
2. Native format - supported over http
3. Native API - clickhouse-go uses this and it means "ClickHouse API" vs "sql/db" API - we need to rename.

In this case, these docs are for the native protocol and native format. I think "native protocol" is fine - although strictly it supports multiple formats e.g. RowBinary